docs/standards/smart-contracts/lsp1-universal-receiver-delegate-up.md

```diff
@@ -36,7 +36,7 @@ function supportsInterface(bytes4 interfaceId) public view returns (bool)
 constructor()
 ```
 
-Registers the **[LSP1UniversalReceiverDelegate interface id](./interface-ids.md)**.
+Registers the **[LSP1UniversalReceiverDelegate InterfaceId](./interface-ids.md)**.
```

Comment:
Suggested change to stick to the convention of function parameters with backticks.

```suggestion
Registers the [**LSP1UniversalReceiverDelegate** `interfaceId`](./interface-ids.md)**.
```